snapcraft/plugins/kbuild.py

```diff
@@ -112,16 +122,60 @@ def __init__(self, name, options, project):
         if logger.isEnabledFor(logging.DEBUG):
             self.make_cmd.append('V=1')
 
+    def unroll_ubuntu_config(self, config_path):
+        try:
+            env = open('debian/debian.env', 'r').read()
```

Comment:
you are not closing `env`, for this to be taken care of read it in a context like this:

```python
with open(os.path.join('debian', 'debian.env'), 'r') as f:
    env = f.read()
```